app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -680,6 +691,10 @@ struct DeviceDelegateOculusVR::State {
     aHeight = scale * (uint32_t)(vrapi_GetSystemPropertyInt(&java, VRAPI_SYS_PROP_SUGGESTED_EYE_TEXTURE_HEIGHT));
   }
 
+  bool Is6DOF() const {
+    return deviceType >= VRAPI_DEVICE_TYPE_OCULUSQUEST_START && deviceType >= VRAPI_DEVICE_TYPE_OCULUSQUEST_END;
```

Comment:
is there a better way to feature detect 6DoF?

Reply:
@MortimerGoro Sorry for waiting. Here is a small bug makes Oculus device can't recognize this is a 6Dof one, You should replace it with `return deviceType >= VRAPI_DEVICE_TYPE_OCULUSQUEST_START && deviceType <= VRAPI_DEVICE_TYPE_OCULUSQUEST_END;`, then we are good to merge it. Good job!.

@cvan They have a new API that is called `GetTrackedRemoteCaps()`. We can use it to improve this function. I will work on this next week.